cassandra/connection.py

```diff
@@ -797,6 +798,7 @@ def __init__(self, host='127.0.0.1', port=9042, authenticator=None,
         self._socket_writable = True
         self.orphaned_request_ids = set()
         self._on_orphaned_stream_released = on_orphaned_stream_released
+        self._application_info = application_info
 
```

Comment:
I asked for this to be declared as an attribute here: https://github.com/scylladb/python-driver/pull/486#discussion_r2131833761
So I'm asking again.

Reply:
for what ?